### PR TITLE
HOCS-4282: remove exception for no standard line for topic

### PR DESCRIPTION
**Pull request by gary-newbolt**

### CODE REVIEW

File: src/main/java/uk/gov/digital/ho/hocs/info/api/StandardLineResource.java [hunk 1]
```diff
@@ -41,10 +42,13 @@ public StandardLineResource(StandardLineService standardLineService) {
         return ResponseEntity.notFound().build();
     }
 
-    @GetMapping(value = "/topic/{topicUUID}/standardLine", produces = APPLICATION_JSON_UTF8_VALUE)
+    @GetMapping(value = "/topic/{topicUUID}/standardLine", produces = APPLICATION_JSON_VALUE)
```
Comment (LiamMacP) [inline src/main/java/uk/gov/digital/ho/hocs/info/api/StandardLineResource.java:45]: While here, might be an idea just ripping usages of `APPLICATION_JSON_UTF8_VALUE` out...

Comment (gary-newbolt) [reply src/main/java/uk/gov/digital/ho/hocs/info/api/StandardLineResource.java:45]: yeah good point.

File: src/main/java/uk/gov/digital/ho/hocs/info/api/StandardLineResource.java [hunk 2]
```diff
@@ -41,10 +42,13 @@ public StandardLineResource(StandardLineService standardLineService) {
...
     public ResponseEntity<GetStandardLineResponse> getStandardLinesForPrimaryTopic(@PathVariable UUID topicUUID) {
         StandardLine standardLine = standardLineService.getStandardLineForTopic(topicUUID);
-        return ResponseEntity.ok(GetStandardLineResponse.from(standardLine));
+        if (standardLine != null) {
```
Comment (LiamMacP) [inline src/main/java/uk/gov/digital/ho/hocs/info/api/StandardLineResource.java:48]: I'm fine if this doesn't get done but would this be better to flip this. Have the expected behaviour be the default return? Return earlier if not found? Just be easier to read the flow in my opinion...

**Review by LiamMacP — APPROVED** [review]
Looks good to me, couple of other small things but not blocking...

File: src/main/java/uk/gov/digital/ho/hocs/info/domain/exception/ApplicationExceptions.java [hunk 3]
```diff
@@ -76,24 +76,6 @@ public LogEvent getEvent() {
...
-        private final LogEvent event;
-
-        public StandardLineNotFoundException(String msg, LogEvent event, Object... args) {
-            super(String.format(msg, args));
-            this.event = event;
-        }
-
-        public StandardLineNotFoundException(String msg, Object... args) {
-            super(String.format(msg, args));
-            this.event = LogEvent.STANDARD_LINE_NOT_FOUND;
```
Comment (LiamMacP) [inline src/main/java/uk/gov/digital/ho/hocs/info/domain/exception/ApplicationExceptions.java:89]: Can this LogEvent be removed now from the enum?

File: src/main/java/uk/gov/digital/ho/hocs/info/api/StandardLineResource.java (hunk 2 above)
Comment (gary-newbolt) [reply src/main/java/uk/gov/digital/ho/hocs/info/api/StandardLineResource.java:48]: Don't particularly mind either way... I've changed given I was there changing stuff.

File: src/main/java/uk/gov/digital/ho/hocs/info/domain/exception/ApplicationExceptions.java (hunk 3 above)
Comment (gary-newbolt) [reply src/main/java/uk/gov/digital/ho/hocs/info/domain/exception/ApplicationExceptions.java:89]: yep... thanks

**Pull request merged**